longTest/LeaderModule_test.go

```diff
@@ -4,12 +4,14 @@ import (
 	"bytes"
 	"github.com/FactomProject/factom"
 	"github.com/FactomProject/factomd/fnode"
+	"github.com/FactomProject/factomd/state"
 	. "github.com/FactomProject/factomd/testHelper"
 	"testing"
 )
 
 //  Currently just tests FCT/EC Commit/Reveal messages to make sure leader is working
 func TestLeaderModule(t *testing.T) {
+	state.EnableLeaderThread = true // enable threaded leader behavior for this test
```

Comment:
unrelated to this pr - but made this change to allow toggle of old/new leader thread behavior during the refactor phase of wax